releasenotes/notes/dogstatsd-origin-detected-unified-98897086225b0783.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    dogstatsd: Implement unified origin detection
```

Comment:
**💬 suggestion:** ‏should we add more precision here. For instance we can mention the new option and what it does.

Reply:
Sure! I'll add some context 👍

Reply:
Done!